models/quote.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import (
+    Boolean,
+    Column,
+    DateTime,
+    Enum,
+    ForeignKey,
+    Integer,
+    String,
+    func,
+)
+from sqlalchemy.orm import relationship
+
+from models.models import Base, auto_str
+
+__all__ = ["Quote"]
+
+
+@auto_str
+class Quote(Base):
```

Comment:
Consider making some factory methods to ease construction, such asa one for creating with an id and one with a string.